mission/edit_me/briefing/situation.html

```diff
@@ -0,0 +1,15 @@
+<font color='#D81717'>ENEMY FORCES:</font>
+INSERT ENEMIES HERE
+INSERT ENEMY AMLCOA (Most Likely Course of Action, AKA, what they're doing, how they act, how they'll act towards us, what we know, etc)
+
+<font color='#1D69F6'>FRIENDLY FORCES:</font>
+Synixe Contractors, INSERT FRIENDLIES HERE
+
+<font color='#139120'>INDEPENDENT FORCES:</font>
+INSERT INDEPENDENT FORCES IF THERE ARE ANY, OTHERWISE REMOVE THIS LINE AND ONE ABOVE
+
+<font color='#993399'>CIVILIAN CONSIDERATIONS:</font>
+INSERT THINGS TO CONSIDER ABOUT CIVILIANS, PRESENCE/ABSENCE AND/OR BEHAVIOUR/SUPPORT TO FRIENDLIES OR ENEMIES
+
+<font color='#663300'>TERRAIN CONSIDERATIONS:</font>
+INSERT ANYTHING YOU FIND RELEVANT ABOUT HOW THE TERRAIN MAY BE ADVANTAGEOUS OR DISADVANTAGEOUS
```

Comment:
I think we should add some form of CONTEXT or proper SITUATION section to inform players of the 'world' they are entering; something that speaks about how this 'world' existed before players arrived.

MISSION tells players what to do, SITUATION tells players what each of the parties involved is like, but I feel like it should also provide context on what everything is standing on; the mission and parties involved don't exist in thin air.